PeerioTechnologies/peerio-mobile
# fix: file name dialog width [branch ch18416]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/18416/mobile-file-name-dialog-text-entry-box-badly-sized-for-long-file-names
#### Testing instructions  
Verify that the width does not extend outside of the view on both android and ios.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
